apps/web/src/app/admin/cohorts/[id]/page-client.tsx

```
@@ -921,33 +1051,164 @@ export function CohortDetailPageClient({
+								</Button>
+							</div>
+						) : (
+							<div className="grid gap-2 lg:grid-cols-2">
+								{cohortWithSessions?.weekly_sessions?.map(
+									(session: any) => (
+										<div
+											key={session.id}
+											className="group relative cursor-pointer overflow-hidden rounded-lg border bg-card transition-all duration-200 hover:shadow-md"
+											onClick={() => handleEditSession(session)}
+										>
+											{/* Day and Time Header */}
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)